front/src/modules/trainschedule/components/Timetable/TimetableTrainCard.tsx

```diff
@@ -13,11 +13,12 @@ import { TrainScheduleValidation, osrdEditoastApi } from 'common/api/osrdEditoas
 
 import { ScheduledTrain } from 'reducers/osrdsimulation/types';
 import RollingStock2Img from 'modules/rollingStock/components/RollingStock2Img';
-import { updateTrainScheduleIDsToModify } from 'reducers/osrdconf';
 import { useDispatch } from 'react-redux';
 import { jouleToKwh, mToKmOneDecimal } from 'utils/physics';
 import { setFailure, setSuccess } from 'reducers/main';
 import { updateSelectedProjection, updateSelectedTrainId } from 'reducers/osrdsimulation/actions';
+import { useOsrdActions } from 'common/osrdContext';
+import { operationalStudiesConfSliceActionsType } from 'reducers/osrdconf/operationalStudiesConf';
```

Comment:
Could you add `type` to all other type imports in the file please :)
```suggestion
import type { operationalStudiesConfSliceActionsType } from 'reducers/osrdconf/operationalStudiesConf';
```

Reply:
Done